cloudcomposerdiff/src/cloudcomposerdiff/__main__.py

```diff
@@ -0,0 +1,111 @@
+"""
+ Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ """
+
+import click
+from google.cloud.orchestration.airflow import service_v1
+
+from cloudcomposerdiff.lib.comparator import EnvironmentComparator
+from cloudcomposerdiff.lib.service import GCPComposerService
+from cloudcomposerdiff.lib.strategies.diffairflowconfig import DiffAirflowConfig
+from cloudcomposerdiff.lib.strategies.diffenvimage import DiffEnvImage
+
+from . import __version__
+
+
+@click.option(
+    "--env1_project_id",
+    type=str,
+    help="GCP Project ID for Cloud Composer Environment #1",
+    required=True,
+)
+@click.option(
+    "--env1_location",
+    type=str,
+    help="GCP location for Cloud Composer Environment #1",
+    required=True,
+)
+@click.option(
+    "--env1_name",
+    type=str,
+    help="GCP name for Cloud Composer Environment #1",
+    required=True,
+)
+@click.option(
+    "--env2_project_id",
+    type=str,
+    help="GCP Project ID for Cloud Composer Environment #2",
+    required=True,
+)
+@click.option(
+    "--env2_location",
+    type=str,
+    help="GCP location for Cloud Composer Environment #2",
```

Comment:
```suggestion
    help="Compute Engine region for Cloud Composer Environment #2",
```

^ if it's zone, change to zone instead